src/sync.ts

```diff
@@ -1,51 +1,39 @@
-import { loadProject } from "@webstudio-is/http-client";
```

Comment:
why is it being removed? the idea is that http-client is ALWAYS used to access the API from both: saas deployment and CLI ...

Reply:
No clue, got this from my upstream branch